packages/@o3r/rules-engine/src/components/rules-engine/ruleset-history/ruleset-history-pres.template.html

```diff
@@ -58,7 +58,7 @@ <h4 class="mb-4">Ruleset Execution History</h4>
                 <div>
                   <ng-container *ngIf="ruleEvaluation.error">
                     <span class="ruleset-panel-category-title">Error:</span>
-                    <pre class="ruleset-panel-category-body error">{{ruleEvaluation.error | json}}</pre>
+                    <pre class="ruleset-panel-category-body error">{{ruleEvaluation.error}}</pre>
```

Comment:
Yeah it is technically an any type, but I couldn't find a case were the error is not an exception which is then stringified with the toString(). I see that the json pipe was set to fix the issue I am actually adressing here so I don't know if this is really useful.
The thing with the json pipe is that it will escape the " " in the error text message and that makes it difficult to read.
What do you think ? 
Should we modify the interface of the debug message to match string only (but it is technically a change of exposed interface), ?
Should we leave the jsonify of the string (and accept messages like "The fact \"myFact\" is missing" ) ?
Should we do our own pipe to stringify both the object and the string ?

Reply:
Hmmm,
Actually changing to `string` is technically a breaking change but keeping `any` but supporting only `string` as display is also not a good solution (neither future proof, if we send any object as error in the future).
I wonder if the best compromise is not your last one and create our own pipe to display the error as we want.
What do you think?